azure-servicebus/README.rst

```diff
@@ -1,222 +1,83 @@
-Microsoft Azure SDK for Python
-==============================
+Microsoft Azure Service Bus SDK for Python
+==========================================
 
-This is the Microsoft Azure Service Bus Runtime Client Library.
+This is the Microsoft Azure Service Bus Client Library.
+This package has been tested with Python 2.7, 3.4, 3.5, 3.6 and 3.7.
 
-This package has been tested with Python 2.7, 3.3, 3.4 and 3.5.
+Microsoft Azure Service Bus supports a set of cloud-based, message-oriented middleware technologies including reliable message queuing and durable publish/subscribe messaging.
 
-For a more complete set of Azure libraries, see the `azure <https://pypi.python.org/pypi/azure>`__ bundle package.
+* `SDK source code <https://github.com/Azure/azure-sdk-for-python/tree/master/azure-servicebus>`__
+* `SDK reference documentation <https://docs.microsoft.com/python/api/overview/azure/servicebus/client?view=azure-python>`__
```

Comment:
How about a third bullet with link to [Service Bus documentation](https://docs.microsoft.com/azure/service-bus-messaging/):

```
* `Service Bus documentation <https://docs.microsoft.com/azure/service-bus-messaging/>`
```
I'll add this to the guidance doc.